caas/kubernetes/provider/bootstrap.go

```diff
@@ -1109,14 +1115,14 @@ func (c *controllerStack) buildContainerSpecForController(statefulset *apps.Stat
 				"--ssl",
 				"--sslAllowInvalidHostnames",
 				"--sslAllowInvalidCertificates",
-				fmt.Sprintf("--sslPEMKeyFile=%s/%s", c.pcfg.DataDir, c.fileNameSSLKey),
+				fmt.Sprintf("--sslPEMKeyFile=%s", c.pathJoin(c.pcfg.DataDir, c.fileNameSSLKey)),
```

Comment:
Do we need to use pathJoin() here? (and below)
The code already uses "/".
I guess for consistency with the other places.

Reply:
This change is just making it generic.
Just thinking if we want to bootstrap to windows in the future then no need to go back here fix it again